grafana/dashboards/orchestrator-overview.json

```diff
@@ -291,13 +306,18 @@
       "pluginVersion": "8.3.3",
       "targets": [
         {
-          "expr": "sum(livepeer_current_sessions_total{node_type=\"orch\"}) / sum(livepeer_max_sessions_total{node_type=\"orch\"}) * 100",
+          "datasource": {
```

Comment:
I wonder why this `datasource` part was added this time with the dashboard export, but was not here before?